tests/sqlparser_common.rs

```diff
@@ -5576,6 +5580,40 @@ fn parse_table_function() {
     );
 }
 
+#[test]
+fn parse_table_valued_function_with_alias_and_column_defs() {
```

Comment:
```suggestion
fn parse_select_with_alias_and_column_defs() {
```
If I follow correctly this is only testing the column defs functionality and the `FROM T` part hasn't changed with this PR so that it can be any table factor?

Reply:
Correct. I updated the function name as per your suggestion.